packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: [">=0.7.0", "<1.1.0"]
+    version: 1.1.1
```

Comment:
Did we find an alternative to `dbt deps`, how did we ensure the package was updated to the specified version?

Reply:
I did not, I ended up installing dbt and running `dbt deps`

Reply:
Can you try running the package in a different dbt project, please?
Use can your commit id as the version of snowflake utils package in the other dbt project. 
When it runs successfully, please take a screenshot and add it to the ticket as a means of validation :)

Reply:
Done! Validation screenshot added into original post :)

Reply:
Great! LGTM :)

Reply:
one small comment, otherwise LGTM!